storage/spanstore/interface.go

```diff
@@ -50,6 +50,13 @@ type Reader interface {
 	FindTraceIDs(ctx context.Context, query *TraceQueryParameters) ([]model.TraceID, error)
 }
 
+// TraceGetParameters contains parameters of a trace get.
+type TraceGetParameters struct {
```

Comment:
the name reads weird
```suggestion
type GetTraceParameters struct {
```

Reply:
I'm trying to follow the silimar naming pattern here:
- TraceQueryParameters
- OperationQueryParameters

These are like <Target>-<Verb>-Parameters

Reply:
GetTraceParameters is the name used in the IDL